src/common/commonutils/SshUtils.c

```diff
@@ -300,16 +321,23 @@ static int CheckSshOptionIsSetToInteger(const char* option, int* expectedValue,
 
 int CheckSshClientAliveInterval(char** reason, void* log)
 {
+    const char* clientAliveInterval = "clientaliveinterval";
     int actualValue = 0;
-    int status = CheckSshOptionIsSetToInteger("clientaliveinterval", NULL, &actualValue, reason, log);
+    int status = 0; 
     
-    if ((0 == IsSshServerActive(log)) && (actualValue <= 0))
+    if ((0 == (status = CheckSshOptionIsSetToInteger(clientAliveInterval, NULL, &actualValue, reason, log))) && (actualValue <= 0))
     {
         OsConfigLogError(log, "CheckSshClientAliveInterval: 'clientaliveinterval' is not set to a greater than zero value in SSH Server response (but to %d)", actualValue);
         OsConfigCaptureReason(reason, "'clientaliveinterval' is not set to a greater than zero value in SSH Server response (but to %d)",
             "%s, also 'clientaliveinterval' is not set to a greater than zero value in SSH Server response (but to %d)", actualValue);
         status = ENOENT;
     }
+    else if (reason)
+    {
+        FREE_MEMORY(*reason);
+        *reason = FormatAllocateString("%sThe %s service reports that '%s' is set to '%d' (that is greater than zero)", 
```

Comment:
For the displayed reason, I chose to have sshd logged without the ''.  In log traces, using '' as clear delimiters to the strings.

Reply:
I am referring to the usage of `'%d'` as opposed to `%d` (which is in other places of the code).

Reply:
Understood. Answer: this is for consistence with all other reported reasons for SSH. Since integers are also read as string values. The '' wrap the SSH configuration option values.

Reply:
I see!